features/find_car.feature

```diff
@@ -0,0 +1,286 @@
+Feature: Find car
+  As a user, I want to use search filters, because they give a quick and easy way to find a car that suits me.
```

Comment:
No `because`. You should use next pattern `As a ...., I want to ..., so I can ...`